src/JustSaying/Fluent/MessagingConfigurationBuilder.cs

```diff
@@ -352,6 +363,68 @@ public MessagingConfigurationBuilder WithRegions(IEnumerable<RegionEndpoint> reg
             return this;
         }
 
+        /// <summary>
+        /// Specifies the <see cref="IDefaultTopicNamingConvention"/> to use.
+        /// </summary>
+        /// <param name="namingConvention">The <see cref="IDefaultTopicNamingConvention"/> to use.</param>
+        /// <returns>
+        /// The current <see cref="MessagingConfigurationBuilder"/>.
+        /// </returns>
+        /// <exception cref="ArgumentNullException">
+        /// <paramref name="namingConvention"/> is <see cref="null"/>.
+        /// </exception>
+        public MessagingConfigurationBuilder WithDefaultTopicNamingConvention(IDefaultTopicNamingConvention namingConvention)
```

Comment:
I honestly find a bit weird that we have to pass a `IDefaultTopicNamigConvention` to this method that should impliciting using the _default topic naming convention_ 😕 ... unless _default_ = _custom_ (meaning there is one that calculates the naming based on a specific logic) which is confusing as well 😄

Reply:
Yeah I can see the confusion - it is indeed custom but the intention is for it to be used whenever `ForTopic<T>()` or `ForQueue<T>()` are invoked without a queue name provided which lets the consumer know it's using the default...not sure how to relay that more clearly 🤔

Reply:
Or we just call it `ITopicNamingConvention` and `IQueueNamingConvention` as @slang25 mentioned before and change the comments accordingly? This feels a bit clearer and we can convey that if you don't provide a queue / topic name then the default implementation of either the two interfaces will be used

Reply:
yeah I would prefer that